lib/services/HDInsight2/lib/hDInsightManagementClient.js

```diff
@@ -276,7 +276,7 @@ var ClusterOperations = ( /** @lends ClusterOperations */ function() {
     
     // Set Headers
     httpRequest.headers['Content-Type'] = 'application/json';
-    httpRequest.headers['User-Agent'] = 'ARM SDK Node v0.2.0';
+    httpRequest.headers['User-Agent'] = 'ARM SDK Node v0.2.1';
```

Comment:
once you update the version in package.json. Please make sure that the same version is reflected over here.